src/Views/components/Header.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import './Header.css';
+
+export default function Header() {
+  return (
+    <div className="header">
+      <ul>
+        <li>
+          <NavLink exact to = '/'>
+                        Home
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to = '/Families'>
+                        Families
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to = '/Movies'>
+                        Movies
+          </NavLink>
+        </li>
+        <li>
+          <NavLink exact to = '/Directors'>
```

Comment:
To prevent navigation issues, it's always a good idea to match the casing on your '/(route)' with that of the App.js page.